source/functions/initializeWecSim.m

```diff
@@ -251,8 +258,8 @@
 if ~isempty(idx)
     for kk = 1:length(idx)
         it = idx(kk);
-        body(it).hydroForcePre(waves.omega,waves.direction,simu.cicTime,waves.bem.count,simu.dt,...
-            simu.rho,simu.gravity,waves.type,waves.waveAmpTime,simu.stateSpace,simu.b2b);
+        body(it).hydroForcePre(waves(1).omega,waves(1).direction,simu.cicTime,waves(1).bem.count,simu.dt,...
```

Comment:
@salhus Moderate comment: Is there a reason that waves(1) is hard coded to the first wave instance? This seems to indicate that you cannot mix and match certain wave features. This relates to my request on documentation to state what combinations of sea states are possible.

Reply:
@nathanmtom 
Yes that is correct. I think this can be expanded and I am planning on enabling mix-n-match in next FY.
I will update documentation to reflect this limitation.